modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -1,112 +1,161 @@
 +
-* PEM
-* JKS
-. In the Certificate File field, click Choose File, and select your certificate file to upload.
+** PEM
+** JKS
+* *Certificate File* -  Click *Choose File*, and select the certificate file to upload.
 +
-If you are uploading a JKS certificate, you also need to provide the passphrase for this certificate and to select its alias from the drop-down menu.
+If you are uploading a JKS certificate, you also need to provide the passphrase for this certificate and select its alias from the drop-down menu.
+. Click *Save*.
 
-== Adding a Certificate Pinset
+== Add a Certificate Pinset
 
-. In the secret group to which you want to add this secret, select Certificate Pinset, and click the Add Certificate Pinset button.
-. In the Name field, write a name for your ceritifcate pinset.
-. In the Certificate File field, click Choose File and select your certificate file to upload.
-// _COMBAK: What type of certifiate to upload?
+. In the *Secret Groups* list view, select the secret group to which to add a certificate pinset. 
+. Select *Certificate Pinset* in the menu on the left, and click *Add Certificate Pinset*. 
+. In the *Certificate Pinset* screen, add the required information. +
+* *Name* - Enter a name for the certificate pinset. 
+* *Certificate File* - Click *Choose File* and select the certificate file to upload.
+* *Expiration Date* (Optional) - Select the expiration date for the certificate.
+// _COMBAK: What type of certificate to upload?
 
-== Adding a Shared Secret
+== Add a Shared Secret
 
-. In the secret group to which you want to add this secret, select Shared Secret, and click the Add Shared Secret button.
-. In the Name field, write a name for your shared secret.
-. In the Type field, select the shared secret type from the drop-down menu.
+. In the *Secret Groups* list view, select the secret group to which to add a shared secret. 
+. Select *Shared Secret* in the menu on the left, and click *Add Shared Secret*. 
+. In the *Add Shared Secret* screen, add the required information. 
 +
-* Username Password: +
-If you select this type, you need to provide a username and password
-* Symetric Key: +
-If you select this type, you need to provide a Base64 string containing symmetric key.
-* S3 Credential: +
-If you select this type, you need to provide the access key ID and the secret access key to an S3 bucket.
-* Blob: +
-If you sleect this type, you need to provide a base64-encoded value.
+* *Name* - Enter a name for your shared secret.
+* *Type* - Select the shared secret type from the drop-down menu.
+ ** *Username Password* - Provide a username and password.
+ ** *Symmetric Key* - Provide a Base64 string containing symmetric key.
+ ** *S3 Credential* - Provide the access key ID and the secret access key to an S3 bucket.
+ ** *Blob* - Provide a base64-encoded value.
+. Click *Save*. 
 
-== Adding a Certificate Revocation List Distribution Configuration
+== Add a Certificate Revocation List Distributor Configuration
```

Comment:
I try to break up noun phrases when they get too long. Add a List of Certificate Revocations for CRL Distribution Points (?).